packages/amplify-graphql-schema-generator/vpc-db-lambda/src/connection.ts

```diff
@@ -7,7 +9,10 @@ export const establishDBConnection = (config: any): any => {
     port: config.port,
     user: config.username,
     password: config.password,
-    ssl: { rejectUnauthorized: false },
+    ssl: {
+      rejectUnauthorized: true,
+      ca: getRDSCertificate(),
+    },
```

Comment:
This code gets executed only if the provided host belongs to RDS. So there is no need to check if it is a RDS host or not.